docs/rewards/rewards-troubleshooting.mdx

```diff
@@ -7,20 +7,125 @@ slug: /project/rewards-troubleshooting
+2. Then, follow the instructions for [joining Arbitrum Sepolia and see your $WXM tokens](/wallet/how-to-join-arbitrum-and-see-your-tokens#add-arbitrum-sepolia-network-to-metamask).
 
-### Sensor Problems
+Once you've completed the above steps, you'll be all set to claim your rewards. Check back tomorrow and see "[How to Claim Rewards](/project/rewards-claiming)" for further guidance.
 
-### Weather Data Gaps
+<hr/>
 
-### Bad Station Deployment
+## No Station Data
 
-### No location Data
+It seems that your station sent no data at all. 
+That either means that your station was offline or not connected to the network. 
 
-### Location Not Verified
+Here's what you need to do:
 
+### For the WS-1000 (WiFi/M5 version)
 
-### User Relocation Penalty
\ No newline at end of file
+1. Check if the station's battery level is low through mobile app settings.
+If the battery is low, the connection between your station and the M5 gateway will be unstable, resulting in data loss. 
+2. Follow the Troubleshooting guide regarding [No WiFi connection](/wifi-m5-bundle/m5-troubleshooting#no-wifi--connection).
+3. Check that your station connection (RF) to the gateway is stable by following the guide [here](/wifi-m5-bundle/m5-troubleshooting#weather-station-not-reporting-data-to-m5-miner-there-are-dashes----on-the-display-console).
+4. Also read through the guide to add the included external antenna [here](/wifi-m5-bundle/m5-troubleshooting#my-station-does-not-connect-to-the-m5-miner).
+
+### For the WS-2000 (Helium version)
+
+1. Check if the station's battery level is low through mobile app settings.
+2. Check for [Helium coverage](https://docs.weatherxm.com/helium-bundle/helium-troubleshooting#how-to-see-if-there-is-helium-coverage-in-your-area) in your area.
+
+<hr/>
+
+## Sensor Problems
+
+Weather data packets failed to pass our quality checks. We evaluate all data packets sent by our weather stations using our QoD (Quality of Data) algorithm. You can read more about this algorithm [here](https://docs.weatherxm.com/project/quality-of-data).
+There are several reasons that you may experience this error, which includes:
+
+- Out of bounds data
+- Spikes on data 
+- Constant values
+- Frozen sensor
+
+Check out our [Deployment Examples](/deployment-examples) and [Best Practices](/wxm-devices/wifi-m5-bundle/install-weather-station#best-practices-for-proper-station-installation) to ensure that your station is deployed correctly,
+and observe it for a couple of days before contacting our support team. If you still receive the error, you are advised to contact our [support team](https://support.weatherxm.com) for more guidance.
+<hr/>
+
+## Weather Data Gaps
+Some data gaps were detected during our data quality checks.
+Indoor or outdoor units are not properly connected to the network, or to each other.
+
+Here's what you need to do:
+
+### For the WS-1000 (WiFi/M5 version)
+
+1. Check if the station's battery level is low through mobile app settings.
```

Comment:
As I suggested in the Linear task, before starting that list, we need to include a screenshot like this one [here](https://docs.weatherxm.com/img/wxm-devices/wifi-m5-bundle/m5-ui-new.png), showing the M5 and explaining what each icon at the top right corner means.

Reply:
I have seen what you have suggested, I don't believe it is a dealbreaker.